templates/layout.html

```diff
@@ -1,10 +1,10 @@
 <!doctype html>
-<html xmlns="http://www.w3.org/1999/xhtml" {% if bag('alternatives', this.alt, 'direction') == 'text-right' %}dir="rtl"{% endif %}>
+<html lang="en" xmlns="http://www.w3.org/1999/xhtml" {% if bag('alternatives', this.alt, 'direction') == 'text-right' %}dir="rtl"{% endif %}>
```

Comment:
Here you add the language 'en', but the template will be used also in other languages, so maybe you need to add some template tags with 'if's like the one about the rtl and ltr text at the end of the same line:
` {% if bag('alternatives', this.alt, 'direction') == 'text-right' %}dir="rtl"{% endif %}`
(but of course different, related to the language)